src/components/StudyList/StudyList.js

```diff
@@ -42,10 +51,30 @@ const StudyList = ({studyList, loading, activeView, roles, history}) => {
       </Container>
     );
   }
+  const myStudyList =
+    myProfile && myProfile.studies.edges.length > 0
+      ? myProfile.studies.edges.map(({node}) => node.kfId)
+      : [];
+
+  const conactCollaborators = collaborators => {
+    return collaborators.length > 0
+      ? collaborators
+          .map(({node}) => node.username + node.firstName + node.lastName)
```

Comment:
Maybe best to concat with spaces? Otherwise the search `Dan Kol` would not return anything.
`[node.username, node.firstName, node.lastName].join(' ')`